server/routes/users.routes.js

```diff
@@ -1,13 +1,20 @@
 import express, { Router } from 'express';
-import controllers from '../controllers';
+import { UserController, ProfileController } from '../controllers';
 import { signInAuth, superAdminAuth } from '../utils/users/permissions.util';
-import { tryCatch, idValidator, validateRole } from '../utils';
+import {
+  tryCatch,
+  idValidator,
+  validateRole,
+  validateProfile,
+  imageUpload,
+} from '../utils';
 
-const { UserController } = controllers;
 const { updateUserRole } = UserController;
-
+const { getProfileById, updateProfile } = ProfileController;
 const router = new Router();
 
 router.patch('/role/:id', signInAuth, superAdminAuth, idValidator, validateRole, tryCatch(updateUserRole));
```

Comment:
It would be nice if the middleware are grouped.

Reply:
explain please.